src/contracts/Repository/SearchService.php

```diff
@@ -120,11 +137,12 @@ interface SearchService
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException if query is not valid
      *
      * @param \Ibexa\Contracts\Core\Repository\Values\Content\Query $query
-     * @param array $languageFilter Configuration for specifying prioritized languages query will be performed on.
+     * @param array<string, mixed>|array<int, string> $languageFilter Configuration for specifying prioritized languages query will be performed on.
      *        Also used to define which field languages are loaded for the returned content.
-     *        Currently supports: <code>array("languages" => array(<language1>,..), "useAlwaysAvailable" => bool)</code>
-     *                            useAlwaysAvailable defaults to true to avoid exceptions on missing translations
-     * @param bool $filterOnUserPermissions if true only the objects which the user is allowed to read are returned.
+     *        Currently, supports two syntaxes:
+     *        - `['languages' => [<language_code_string>,…], 'useAlwaysAvailable' => <bool>]`
+     *        - `[<language_code_string>,…]` where `useAlwaysAvailable` defaults to `true` to avoid exceptions on missing translations.
```

Comment:
This doesn't render well in PHPStorm.
![image](https://github.com/user-attachments/assets/6c29ffe3-42bb-4eb7-a130-9b83860b9fb3)

Reply:
By adding a new line before the list, it doesn't change the PHP API Ref rendering and I got this on PHPStorm

![image](https://github.com/user-attachments/assets/b6697021-cf04-4238-9dbb-f06dca2100cc)

Reply:
Surprisingly enough, if you reduce the indent:
```suggestion
     *   - `['languages' => [<language_code_string>,…], 'useAlwaysAvailable' => <bool>]`
     *   - `[<language_code_string>,…]` where `useAlwaysAvailable` defaults to `true` to avoid exceptions on missing translations.
```
it renders as:
![image](https://github.com/user-attachments/assets/0f58dc19-b58d-46e8-988c-11362ef857c0)
which is what I'd expect.

It ain't pretty in PHPDoc, but I prefer better rendering to indent alignment in this particular case.
Does that work for phpDocumentor?